### PR TITLE
Fixed problems with changeSets

### PR DESCRIPTION
**Pull request by polanjiri**
Hi everyone,
I am making these changes to resolve problem with **changeSets**. 

1. Method `get_changeset_items()` was returning empty list.
2. Three methods `get_revision()` / `get_revision_branch()`/ `get_repo_url()` were returning KeyError
```
  File "/Users/poli/Python/jenkinsapi/jenkinsapi/build.py", line 85, in get_revision
    vcs = self._data['changeSet']['kind'] or 'git'
KeyError: 'changeSet'
```

### CODE REVIEW

**Comment by ryan4yin** [comment]
same problem, waiting for this to be merged.

File: jenkinsapi/build.py [hunk 1]
```diff
@@ -160,10 +165,11 @@ def _get_git_rev(self):
         # Sometimes we have None as part of actions. Filter those actions
         # which have lastBuiltRevision in them
         _actions = [x for x in self._data['actions']
-                    if x and "lastBuiltRevision" in x]
+                    if x and "build" in x]
+        print(_actions)
```
Comment (lechat) [inline jenkinsapi/build.py]: This fails build for Python 2.7. Please remove this line.

**Comment by lechat** [comment]
This change break related unit tests. Please fix that.

File: jenkinsapi/build.py (hunk 1 above)
Comment (polanjiri) [reply jenkinsapi/build.py]: @lechat Could you please help. I'm not sure why tests are failing.

**Comment by vladmalynych** [comment]
Failing `Travis` tests seems to be not related to this change:

```
E           jenkinsapi.custom_exceptions.JenkinsAPIException: Problem installing plugin 'aws-java-sdk'.
```

**Pull request merged**